src/components/Signup/SignupContainer.tsx

```diff
@@ -1,10 +1,25 @@
 import { useFormik, FormikProps } from "formik";
-import * as Yup from "yup";
+import { useMutation } from "react-query";
+import { useHistory } from "react-router-dom";
 import { FormValues } from "./types";
 import Signup from "./Signup";
-import { signup } from "../../utils/constants";
+
+import { requestSignup } from "../../utils/apis/signup";
+import { signupValidator } from "../../utils/yups/signup";
 
 const SignupContainer = () => {
+  const history = useHistory();
+  const mutation = useMutation((values: FormValues) => requestSignup(values), {
+    onSuccess: () => {
+      // TODO: 성공했을 경우 회원가입이 완료되었습니다. 문구를 Toast로 띄워 사용자에게 알려준다.
+      alert("회원가입이 완료되었습니다.");
+      history.push("/");
```

Comment:
CI를 구축한 이점을 얻으려면 적어도 CI에서는 아무런 오류 또는 경고가 발생하지 않았다는 것을 보장 받는게 좋습니다. 따라서 린트 내 경고(warning)도 허용하지 않는 게 좋으므로 `// TODO:` 코멘트 주변에 `eslint-disable`을 넣고 왜 disable을 했는지, 추후 처리할 내용이 있다면 어떻게 처리할 것인지 코멘트로 적어주세요~!

예를 들어,

```suggestion
     // TODO: Toast가 완성되면 alert를 제거한다
     // eslint-disable-next-line no-alert
     alert("회원가입이 완료되었습니다.");
     
      history.push("/");
```

Reply:
오호 그렇군요! 짚어주셔서 감사해요 😁